plasma_framework/test/src/utils/RLP.test.js

```diff
@@ -36,8 +47,11 @@ contract('RLP', () => {
         await testNumberDecoded(this.test.decodeUint, 100);
     });
 
-    it('should decode int 0', async () => {
-        await testNumberDecoded(this.test.decodeInt, 0);
+    it('should decode 0x00', async () => {
```

Comment:
It turns out that toUint() should _not_ decode `0x00`.
I'll change this test when I fix that